### PR TITLE
Alpha

### PR DESCRIPTION
**Pull request by akhatun**

### CODE REVIEW

**Review by rolavick — APPROVED** [review]

File: PWGUD/AQC/udQCmidRap.cxx [hunk 1]
```diff
@@ -134,8 +134,8 @@ struct UDQCmid {
 
   // ...............................................................................................................................................
   void processMain(CC const& collision, BCs const& bct0s,
-                   TCs const& tracks, FWs const& fwdtracks, ATs const& /*ambtracks*/, AFTs const& /*ambfwdtracks*/,
-                   aod::FT0s const& /*ft0s*/, aod::FV0As const& /*fv0as*/, aod::FDDs const& /*fdds*/,
+                   TCs const& tracks, FWs const& fwdtracks, ATs const& ambtracks, AFTs const& ambfwdtracks,
+                   aod::FT0s const& ft0s, aod::FV0As const& fv0as, aod::FDDs const& fdds,
```
Comment (vkucera) [inline PWGUD/AQC/udQCmidRap.cxx:138]: Please don't revert fixes.

File: PWGUD/AQC/udQCmidRap.cxx [hunk 2]
```diff
@@ -459,8 +459,8 @@ struct UDQCmid {
 
   //.....................................................................................................................
   // Distribution of number of PV contributors for all collisions and those with empty FT0
-  void processFewProng(CC const& collision, BCs const& /*bct0s*/,
-                       aod::FT0s const& /*ft0s*/, aod::FV0As const& /*fv0as*/, aod::FDDs const& /*fdds*/)
+  void processFewProng(CC const& collision, BCs const& bct0s,
+                       aod::FT0s const& ft0s, aod::FV0As const& fv0as, aod::FDDs const& fdds)
```
Comment (vkucera) [inline PWGUD/AQC/udQCmidRap.cxx:463]: Please don't revert fixes.

**Review by vkucera — COMMENTED** [review]
Please use more informative PR titles.

File: PWGUD/AQC/udQCmidRap.cxx (hunk 2 above)
Comment (akhatun) [reply PWGUD/AQC/udQCmidRap.cxx:463]: There are some issues with this task, unresolved commits from the past creating many conflicts in the master branch which is why this PR is done separately. Working on fixing the conflict in the main branch. Thanks for your comments, will do a new PR with fixes.

**Pull request merged**